linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/oval/shared.xml

```diff
@@ -1,16 +1,15 @@
 <def-group>
-  <definition class="compliance"
-  id="installed_OS_is_certified" version="1">
+  <definition class="compliance" id="installed_OS_is_FIPS_certified" version="1">
     <metadata>
-      <title>Vendor Certified Operating System</title>
+      <title>FIPS 140-2 Certified Operating System</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
         <platform>multi_platform_rhosp</platform>
         <platform>multi_platform_fedora</platform>
```

Comment:
I think Fedora should be removed, not sure about OpenStack though.

Reply:
@yuumasato It doesn't mean that the definition passes on Fedora, it means that the definition is applicable to Fedora. The definition fails on Fedora, it passes only on RHEL6 and RHEL7.

Reply:
This could be `multi_platform_all` then.

Reply:
Actually no, the applicability that is currently set should stay.

Reply:
I don't have a strong opinion on this. I'm okay with the applicability as is now,  but I don't see why this can't apply for other systems, they can also be FIPS certified.